smalltalksrc/VMMaker/Spur32BitMMLESimulator.class.st

```diff
@@ -83,7 +83,7 @@ Spur32BitMMLESimulator >> fetchFloatAt: floatBitsAddress into: aFloat [
 	aFloat at: 1 put: (self uint32AtPointer: floatBitsAddress+4)
 ]
 
-{ #category : #'object access' }
+{ #category : #'as yet unclassified' }
```

Comment:
```suggestion
{ #category : #'object access' }
```

Reply:
This is a bug from the automatic protocol selection I think. We can improve it in the next sprint.